src/control/cmd/dmg/pretty/pool.go

```diff
@@ -303,17 +310,39 @@ func printListPoolsRespVerbose(out io.Writer, resp *control.ListPoolsResp) error
 			continue
 		}
 		table = append(table, poolListCreateRowVerbose(pool))
+
 	}
 
 	fmt.Fprintln(out, formatter.Format(table))
 
 	return nil
 }
 
+// UpdateListPoolsResponse will update the pool which has been rebuild and not in idle state
+func UpdateListPoolsResponse(final_resp *control.ListPoolsResp, resp *control.ListPoolsResp, rebuildOnly bool) error {
```

Comment:
I think this helper function should be in the dmg package and unexported. Modifying the response itself feels a bit outside the scope of the pretty package, which is about formatting output text.